app/views/field_of_operation/show.html.erb

```diff
@@ -1,3 +1,6 @@
+<% content_for :extra_headers do %>
+  <meta name="description" content="<%= strip_tags(@content_item.description) %>">
```

Comment:
I think we still need to include `<%= render "govuk_publishing_components/components/machine_readable_metadata", { content_item: content_item.to_h, schema: :article } %>` in this block, similar to https://github.com/alphagov/frontend/blob/main/app/views/fatality_notice/show.html.erb#L6

Reply:
Thanks, I've added this, though I can't see any difference in the HTML.

Reply:
Oh! Maybe we don't need it anymore then? 🤔 I guess we can revisit and remove it from all the other migrated routes later?

Reply:
Just to double-check: The machine readable metadata is right at the end of the HTML, is that right? So is it possible something else is adding it in this case (and either adding or not adding it in others)?

Reply:
This is the current metadata in the live page:
```
<script src="[/assets/frontend/application-633967b9ddd3282900f0cc893c3bf16fa13b1caa003db4d3cd022d3f02819108.js](https://www.gov.uk/assets/frontend/application-633967b9ddd3282900f0cc893c3bf16fa13b1caa003db4d3cd022d3f02819108.js)" type="module"></script><script type="application/ld+json">
  {
  "@context": "http://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [
    {
      "@type": "ListItem",
      "position": 1,
      "item": {
        "name": "Home",
        "@id": "https://www.gov.uk/"
      }
    }
  ]
}
</script>
```

Reply:
@KludgeKML @deborahchua Oh yeah, ignore me as Keith is right - I was wrongly looking in the `<head>` of the page for the difference in HTML.